json-rpc/docs/type_transaction.md

```diff
@@ -115,6 +115,7 @@ The transaction script and arguments of the script call.
 | type           | string       | Name of the script code, see [transaction script doc](../../language/diem-framework/script_documentation/script_documentation.md) for all available script names. If a script function, this field is set to the string `"script_function"`. |
 | code           | string       | Hex-encoded compiled move script bytes        |
 | arguments      | List<string> | List of string value of the script arguments. Contains type information. |
+| arguments_bcs  | List<string> | List of hex-encoded string of BCS bytes representing script function arguments. Does not contain type information. |
```

Comment:
could add words to tell only script function will have arguments_bcs value, and the `arguments` will not be rendered if it is a script function